features/individual_charity_pages/charity_page.feature

```diff
@@ -61,6 +61,18 @@ Feature: Web page owned by each charity
     Then I should not see "Health" within "what_they_do"
     And I should not see "Education" within "what_they_do"
     And I should not see "Animal Welfare" within "what_they_do"
+    
+  Scenario: display categories as links
+    Then I should see "Health" < tagged > with "a"
+    And  I should see "Education" < tagged > with "a"
+    And  I should see "Voluntary" < tagged > with "a"
+    And  I should see "Finance" < tagged > with "a"
+
```

Comment:
there's a chance to pull the whole scenario to a much more declarative level if we are interested in that ...

http://itsadeliverything.com/declarative-vs-imperative-gherkin-scenarios-for-cucumber

https://danashby.co.uk/2014/12/19/declarative-or-imperative-how-do-you-write-your-feature-file-scenarios/